### DISCUSSION TITLE
CoPilot for GitHub Teams

### DISCUSSION THREAD

**Discussion by pedrooliveiraeco** [community/community#30578] [Codespaces]
Hello,

Just looking for info on how CoPilot works and it's pricing for GitHub Teams. I know I have to enable CodeSpaces for my organization but I'm wondering how it works after that? Do all users get it for free or what are the costs associated with it for the organization? 

Thanks!

**Comment by bdmac** [comment]
CoPilot and Codespaces are two separate things and neither is required for the other. It's unclear which you're asking about here.

**Comment by pedrooliveiraeco** [comment]
Thanks bdmac!

See here: https://docs.github.com/en/enterprise-cloud@latest/codespaces/codespaces-reference/using-github-copilot-in-github-codespaces

Unless I'm reading that wrong (very well could be) I need to enable codespaces to use copilot on my teams organization. Before I do that, I'd like to know if there is pricing information available or if anyone else here uses it.

**Reply by bdmac** [reply]
Yah I think my confusion is that you don't _have_ to use them together and the pricing for CoPilot isn't impacted by whether you're using Codespaces? You only need to enable Codespaces if you are also using Codespaces.

https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot
https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces

**Comment by bochen87** [comment]
Hi, is the copilot available for a team? Currently, you can only subscribe on an individual user basis.
